src/components/ui/UiChartDonut.vue

```diff
@@ -0,0 +1,41 @@
+<script setup lang="ts">
+const series = [35, 15, 5, 45]
```

Comment:
I think we should put this number as props to the component, so that we can use the number from the data.

Reply:
Ok , i will create an component name UiChartDonut just to display information and one component SurfaceChartDonut where we get the data